prov/tcp/src/xnet.h

```diff
@@ -92,6 +93,7 @@ extern int xnet_disable_autoprog;
 extern int xnet_io_uring;
 extern int xnet_max_saved;
 extern size_t xnet_max_inject;
+extern size_t xnet_max_rx_size;
```

Comment:
I'd rename to xnet_buf_size.  This impacts the buffer size for both the send and receive side.  And it technically is the buffer size, not a max.